app/dashboard/templates/bounty/details.html

```diff
@@ -444,9 +444,69 @@ <h5 class="bounty-heading">{% trans "Funder" %}</h5>
   <script src="{% static "v2/js/bootstrap.min.js" %}"></script>
   <script src="{% static "v2/js/clipboard.js" %}"></script>
   <script>
-    var bootstrapTooltip = $.fn.tooltip.noConflict()
-    var bootstrapModal = $.fn.modal.noConflict()
-    $.fn.bootstrapModal = bootstrapModal
+    let bootstrapTooltip = $.fn.tooltip.noConflict()
+    let bootstrapModal = $.fn.modal.noConflict()
+  </script>
+  <script>
+    $.fn.runTooltip = bootstrapTooltip
+
+    $('[data-toggle="popover"]').popover()
+
+    $('body').popover({
+      selector: '.bounty_row',
+      html: true,
+      title: function() {
+        let element = $(this);
+        let data = '';
+          fetch('/api/v0.1/profile/'+ element.attr('data-username'), {
+          method: 'GET'
+          }).then(function(response) {
+            return response.json();
+          }).then(function(json) {
```

Comment:
I think could be better to move this to a separate file to be used in diff places.
Also I will recommend to use string literals instead of looking for each element, will be faster to process and maintain (example https://github.com/gitcoinco/web/pull/4145/files#diff-f0d33abd6c13c2e1306d4e8741433411R499)

Reply:
Also check here this code I think can be helpful so you will fetch just the base or mount all the code with literals in that way you are not going to need to setup all you data and then append to the html 
https://github.com/gitcoinco/web/blob/9b6ededd9c239f2675a5f3b237c5dce11ec50d81/app/assets/v2/js/pages/bounty_details.js#L725